File: exercism/python/darts/darts.py
import math


def score(x, y):
    radius = math.sqrt(x ** 2 + y ** 2)
    if radius <= 1:
        points = 10
    elif radius <= 5:
        points = 5
    elif radius <= 10:
        points = 1
    else:
        points = 0
    return points

File: exercism/java/darts/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double radius = Math.sqrt(Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2));
        if (radius <= 1) {
            return 10;
        }
        if (radius <= 5) {
            return 5;
        }
        if (radius <= 10) {
            return 1;
        }
        return 0;
    }
}
